File: 251110/두 개의 격자 비교하기/compare-two-grid.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n');
const [N, M] = input[0].split(' ').map(Number);
const firstGrid = input.slice(1, N + 1).map(row => row.split(' ').map(Number));
const secondGrid = input.slice(N + 1).map(row => row.split(' ').map(Number));

const resultGrid = [];
for (let i = 0; i < N; i += 1) {
    let row = [];
    for (let j = 0; j < M; j += 1) {
        if (firstGrid[i][j] === secondGrid[i][j]) {
            row.push(0);
        } else {
            row.push(1);
        }
    }
    resultGrid.push(row);
}

resultGrid.forEach(row => console.log(row.join(' ')));

File: trail0/두 개의 격자 비교하기/compare-two-grid.cpp
#include <iostream>
using namespace std;

int main() {
    int N, M;
    cin >> N >> M;
    int first_grid[N][M], second_grid[N][M], new_grid[N][M];
    for (int r = 0; r < N; r++) {
        for (int c = 0; c < M; c++) {
            cin >> first_grid[r][c];
        }
    }
    for (int r = 0; r < N; r++) {
        for (int c = 0; c < M; c++) {
            cin >> second_grid[r][c];
        }
    }
    for (int r = 0; r < N; r++) {
        for (int c = 0; c < M; c++) {
            if (first_grid[r][c] == second_grid[r][c]) {
                new_grid[r][c] = 0;
            } else {
                new_grid[r][c] = 1;
            }
        }
    }
    for (int r = 0; r < N; r++) {
        for (int c = 0; c < M; c++) {
            cout << new_grid[r][c] << ' ';
        }
        cout << endl;
    }
    return 0;
}